plugins/Dashboard/API.php

```diff
@@ -36,15 +37,16 @@ public function __construct(Dashboard $dashboard, Model $model)
      *
      * @return array[]
      */
-    public function getDashboards($login = '', $returnDefaultIfEmpty = true)
+    public function getDashboards($login = '', $returnDefaultIfEmpty = true )
     {
         $login = $login ? $login : Piwik::getCurrentUserLogin();
+        $idSite = Common::getRequestVar('idSite');
```

Comment:
This should be a method parameter instead. Maybe we could use `0` as default (which would be `all websites`)